UnityProjectDP/Assets/Scripts/Visualization/UI/PopUps/MethodPopUp.cs

```diff
@@ -25,31 +58,46 @@ public override void Confirmation()
                 return;
             }
 
-            var method = new Method
+            var newMethod = new Method
             {
                 Name = inp.text,
                 ReturnValue = dropdown.options[dropdown.value].text,
                 arguments = _parameters
             };
-            ClassEditor.AddMethod(className.text, method, ClassEditor.Source.editor);
-
+            if (_formerName == null)
+                ClassEditor.AddMethod(className.text, newMethod, ClassEditor.Source.editor);
+            else
+            {
+                ClassEditor.UpdateMethod(className.text, _formerName, newMethod);
+                _formerName = null;
+            }
```

Comment:
nech to je symetricky 
```suggestion
            if (_formerName == null)
            {
                ClassEditor.AddMethod(className.text, newMethod, ClassEditor.Source.editor);
            }
            else
            {
                ClassEditor.UpdateMethod(className.text, _formerName, newMethod);
                _formerName = null;
            }
```